WalletWasabi.Backend/wwwroot/faq.html

```diff
@@ -0,0 +1,400 @@
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="contribution.html">Contribution</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="about.html">About us</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="https://blog.wasabiwallet.io" target="_blank">Blog</a>
+                    </li>
+                    <li class="nav-item active">
+                        <a class="nav-link" href="faq.html">Help</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="btn btn-secondary" href="index.html#download">Download</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <div class="fixed-img">
+        <img src="img/04_support.png" alt="Wasabi Wallet - Help">
+    </div>
+
+    <div class="content">
+        <section class="blured-section p-100">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-8 offset-md-2">
+                        <h1 class="text-center title-lined mb-5">FAQ</h1>
+
+                        <div class="card">
+                            <input type="checkbox" id="faq1">
+                            <label for="faq1">
+                                WHAT IS A “coinjoin”?
+                            </label>
+                            <div class="card-content">
+                                <p>Coinjoin is a mechanism by which multiple participants combine their coins (or UTXOs) into one large transaction with multiple inputs and multiple outputs. An observer cannot determine which output belongs to which input, and neither can the participants themselves. This makes it difficult for outside parties to trace where a particular coin originated from and where it was sent to (as opposed to regular bitcoin transactions, where there is usually one sender and one receiver).</p>
+
+                                <p>This can be done with non-custodial software like Wasabi Wallet that eliminates the risk of funds disappearing or being stolen. Each of the signatures are created on the participants’ computers, so anyone trying to connect the signatures is unable to alter the transaction or redirect the funds. The funds will always be in a Bitcoin address that you control. </p>
+
+                                <p>It’s possible to do this in a decentralized way so that the service does not rely on external parties or centralized servers. It just needs the participants of the transaction. </p>
+
+                                <p>coinjoin can be applied multiple times, and as many transactions are grouped together, participants may save on transaction fees. coinjoin is the preferred method of gaining privacy in the Bitcoin network.</p>
```

Comment:
```suggestion
                                <p>Eventhough users get a decent level of privacy after only one round of coinjoin, the process can be repeated multiple times, increasing the privacy substantially. Because it is very effective and relatively cheap, coinjoin is the preferred method of gaining privacy in the Bitcoin network.</p>
```